src/Tutor.Infrastructure/Database/DataImport/LearnerSqlExporter.cs

```diff
@@ -72,10 +72,10 @@ private static string BuildEnrollmentSql(List<UserLearnerColumns> learners, Doma
                 foreach (var kcId in domainContent.KnowledgeComponents.Select(kc => kc.Id))
                 {
                     sqlBuilder.Append(
-                        "INSERT INTO public.\"KcMasteries\"(\"Id\", \"Mastery\", \"KnowledgeComponentId\", \"LearnerId\", \"IsStarted\", \"IsPassed\", \"IsSatisfied\", \"IsCompleted\", \"HasActiveSession\") VALUES");
+                        "INSERT INTO public.\"KcMasteries\"(\"Id\", \"Mastery\", \"KnowledgeComponentId\", \"LearnerId\", \"IsStarted\", \"IsPassed\", \"IsSatisfied\", \"IsCompleted\") VALUES");
                     sqlBuilder.AppendLine();
                     sqlBuilder.Append("\t(" + ++kcMasteryId + ", 0.00, " + kcId + ", "
-                                       + learnerId + ", false, false, false, false, false);");
+                                       + learnerId + ", false, false, false, false);");
                     sqlBuilder.AppendLine().AppendLine();
```

Comment:
The seeding script needs to be generated again because of this change, but I can't do it since I don't have the Excel files.

Reply:
Notes. I will expand the authoring tools to include @lukaDoric `Course` additions on Monday and update the seed-RS and starting templates accordingly.